src/honeybadger/core.clj

```diff
@@ -37,15 +44,23 @@
   (->> st st/parse-trace-elems (map format-stacktrace-elem)))
 
 (defprotocol Notifiable
+  (error-map-no-causes [this])
```

Comment:
Do you see a compelling reason to expand the protocol? This seems like just a concern of `Throwable`'s implementation of the protocol, not the protocol itself.